app/main/lib/langid.py

```diff
@@ -75,3 +76,24 @@ def langid(text):
   def test():
     cld3.get_language("Some text to check")
     return True
+
+class FastTextLangidProvider:
+# https://fasttext.cc/docs/en/language-identification.html
+  fasttext_model = fasttext.load_model("extra/fasttext_language_id/lid.176.ftz")
```

Comment:
should we host this in s3 like (I believe) we do for other models?

Reply:
I'd like to not do so in this case as it would increase the complexity. With sentence transformers models, we automatically download and populate S3 with the model from hugging face if it doesn't exist, but fasttext doesn't have this fallback and it would have to be written manually, which seems overkill right now to avoid a 917KB file in the repo.